### ISSUE TITLE
Configure Fuseki to fit our Graph Store structure

### ISSUE THREAD

**Issue by blankdots** [ATTX-project/graphmanager#2]
#### Description:
We need to customise Fuseki to fit our configuration of the graph store.
There should be both in-memory and TDB backed services available.

#### DoD:
All the graphs will be merged into default graph via the configuration.
Add ontology data model to Fuseki container by default:
* Design the ontology
* include ontology part of the graph
* include missing ontologies such as provenance, working data, etc.

#### Testing:
BDD, SPARQL test

**Comment by blankdots** [comment]
Depends on: https://github.com/ATTX-project/platform-deployment/issues/9

**Comment by blankdots** [comment]
Documentation and examples:
* https://github.com/kerameikos/framework/wiki/Deploying-Fuseki
* https://github.com/apache/jena/blob/master/jena-fuseki2/examples/service-tdb-mem.ttl
* https://gist.github.com/acoburn/ff8984ab4407dc02962e5db4b4466222
* https://github.com/apache/jena/blob/master/jena-fuseki2/examples/service-inference-2.ttl
* https://gist.github.com/lawlesst/ba32ad4fcc830b32823a
* https://gist.github.com/TimmyStorms/bc2014756dcec8b6b814
* https://jena.apache.org/documentation/tdb/assembler.html
* http://mail-archives.apache.org/mod_mbox/jena-users/201603.mbox/%3CCA%2BRbaMCSPHOO6%3DHTBETY-C1e2VfDEhHRfJ0GMSBfwQ70KEsm3Q%40mail.gmail.com%3E
* https://github.com/mparodi/TripleORM/blob/master/config.ttl

**Comment by blankdots** [comment]
Initial documentation available at: https://github.com/ATTX-project/project-management/wiki/Graph-Store
The admin user password is preset from Gradle build container or following instructions from: https://github.com/stain/jena-docker/tree/master/jena-fuseki
